extensions/amp-consent/0.1/consent-state-manager.js

```diff
@@ -369,10 +362,7 @@ export class ConsentInstance {
         return;
       }
 
-      const value = composeStoreValue(
-        consentInfo,
-        this.isAmpConsentV2ExperimentOn_
-      );
+      const value = composeStoreValue(consentInfo, true);
```

Comment:
Will also need to clean up the `composeStoreValue` method as you mentioned in the PR description.